minimal-notebook/start-notebook.sh

```diff
@@ -12,5 +12,4 @@ if [ ! -z "$GRANT_SUDO" ]; then
 fi
 
 # Start the notebook server
-exec su $NB_USER -c "env PATH=$PATH jupyter notebook $@"
-
+exec su $NB_USER -c "env PATH=$PATH jupyter notebook $*"
```

Comment:
This assumes `IFS` is defined and not null, which is totally sensible. Semantically the difference of before:

```bash
exec su $NB_USER -c "env PATH=$PATH jupyter notebook \"$1\" \"$2\" ..."
```

after:

```bash
exec su $NB_USER -c "env PATH=$PATH jupyter notebook $1 $2..."
```

is that correct?